docs/UserGuide.adoc

```diff
@@ -27,8 +27,7 @@ Looking for an all-in-one solution to manage your recipe? Look no further!
 *CookBuddy* is an integrated platform fully customized for users with the aim of helping you manage your recipes effectively. The application allows you to keep track of your recipes.
 It looks like you will never have to worry about finding your recipe anymore!
 
-All your recipe information is comfortably compartmentalized on our simple and clean Graphical User Interface (GUI) and we are optimized for users who are very comfortable
-working on the Command Line Interface (CLI).
+All your recipe information is stored on our simple and clean Graphical User Interface (GUI) and we are optimized for users who are very comfortable with working on the Command Line Interface (CLI).
```

Comment:
```suggestion
All your recipe information is stored on our simple and clean Graphical User Interface (GUI) and we are optimized for users who are very comfortable with working on the Command Line Interface (CLI).
```

1. A UI can't store information;
2. 'We' aren't optimised; 'CookBuddy is optimised'. 

Hence:
"CookBuddy uses a *g*raphical *u*ser *i*nterface (GUI) to display your recipes, while allowing interaction using text, which is optimised for users familiar with the command-line, and are fast typists."